api/documents/views.py

```diff
@@ -0,0 +1,75 @@
+from http import HTTPStatus
+from typing import List
+
+from django.http import HttpRequest
+from ninja import Router
+from ninja.pagination import LimitOffsetPagination, paginate
+
+from documents.models import Chunk, Document
+from documents.schemas import ChunkIn, ChunkOut, DocumentIn, DocumentOut
+
+router = Router()
+
+@router.post("/chunk/", response={HTTPStatus.CREATED: ChunkOut})
+def create_chunk(request: HttpRequest, payload: ChunkIn):
+    chunk = Chunk(**payload.dict())
+    chunk.full_clean()
+    chunk.save()
+    return HTTPStatus.CREATED, chunk
+
+@router.get("/chunk/", response={HTTPStatus.OK: List[ChunkOut]})
+@paginate(LimitOffsetPagination)
+def list_chunks(request: HttpRequest):
+    return Chunk.objects.all()
+
+@router.get("/chunk/{id}", response={HTTPStatus.OK: ChunkOut})
+def retrieve_chunk(request: HttpRequest, id: int):
+    chunk = Chunk.objects.get(id=id)    
+    return chunk
+
+@router.put("/chunk/{id}", response={HTTPStatus.OK: ChunkOut})
+def update_chunk(request: HttpRequest, id: int, payload: ChunkIn):
+    chunk = Chunk.objects.get(id=id)
+    for attr, value in payload.dict(exclude_unset=True).items():
+        setattr(chunk, attr, value)
+    chunk.full_clean()
+    chunk.save()
+    return chunk
+
+@router.delete("/chunk/{id}", response={HTTPStatus.NO_CONTENT: None})
+def delete_chunk(request: HttpRequest, id: int):
+    chunk = Chunk.objects.get(id=id)
+    chunk.delete()
+    return HTTPStatus.NO_CONTENT, None
```

Comment:
**issue:** this should return `200 OK` (that's the default), not `NO_CONTENT`, for future reference see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status